packages/tldraw/src/lib/ui/hooks/useKeyboardShortcuts.ts

```diff
@@ -140,39 +140,39 @@ export function useKeyboardShortcuts() {
 	}, [actions, tools, isReadonlyMode, editor, isFocused])
 }
 
-// Shift is !
-// Alt is ?
-// Cmd / control is $
-// so cmd+shift+u would be $!u
-
+// The "raw" kbd here will look something like "a" or a combination of keys "del,backspace",
+// or modifier keys (using ! for shift, $ for cmd, and ? for alt). We need to first split them
+// up by comma, then parse each key to get the actual key and modifiers.
 function getHotkeysStringFromKbd(kbd: string) {
 	return getKeys(kbd)
 		.map((kbd) => {
 			let str = ''
-			const chars = kbd.split('')
-			if (chars.length === 1) {
-				str = chars[0]
+
+			const shift = kbd.includes('!')
```

Comment:
i somehow didn't realize that the `!?$` was our own homebrew choices for these action characters. can we change these as long as your updating this code? it's super arcane/difficult to read, especially when trying to update the `actions.tsx` file.

if we can use `⇧ ⌥ ⌘` Unicode symbols that _massively_ help DX here and maintainability

Reply:
IIRC it was due to the weirdness around option / alt and cmd / ctrl. If we can, mind if we do it as a separate PR?

Reply:
yes, please!